content/docs/2.10/scalers/solr.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Solr"
+availability = "v2.10+"
+maintainer = "Community"
+description = "Scale applications based on Solr query results."
+go_file = "solr_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the solr trigger that scales based on the outputs of a Solr query.
+
+```yaml
+triggers:
+  - type: solr
+    metadata:
+      host: "http://solr-service.solr-ns.svc.cluster.local:8983"
+      query: "*:*"
+      core: "my_core"
+      targetQueryValue: "1"
+      activationTargetQueryValue : "3"
+      username: "solr"
+```
+
+**Parameter list:**
+
+- `host` - The hostname for connecting to the Solr service.
+- `query`- A Solr query that should return single numeric value. (Default: `*:*`, Optional)
+- `core` - Your collection name on Solr.
```

Comment:
I'm not familiar with Solr but wouldn't this be a more reasonable name then?
```suggestion
- `collection` - Your collection name on Solr.
```

Reply:
It is definitely more understandable for the user.